### DISCUSSION TITLE
Proposal: new syntax for boolean variants

### DISCUSSION THREAD

**Discussion by GermanJablo** [joe-bell/cva#272] [Ideas]
I would like to propose the following:

```ts
// current
variants: {
   disabled: {
     true: 'opacity-50 bg-gray-500 pointer-events-none'
   }
 }

// proposal
variants: {
   disabled: 'opacity-50 bg-gray-500 pointer-events-none',
 }
```

**Comment by joe-bell** [comment]
Interesting, I really like this. I guess this only works for `true` states though

**Reply by GermanJablo** [reply]
yes, exactly!

**Comment by seb-jean** [comment]
It is not possible to pass CSS classes when a value is `false`?

**Reply by GermanJablo** [reply]
On that occasion you would have to use the current API. What I propose is useful for the large number of times when there are no additional classes in "false".

**Comment by joe-bell** [comment]
I gave this a go and, while I still like the syntax, I don’t think the trade-off is worth it. It ended up adding more complexity to the types and runtime than I’d like for something that saves a couple of lines 😢 

Thanks for suggesting it though!
